Fix minor typo in documentation (#1241)
finos/FDC3: docs/api/spec.md

The role of FDC3 API is to establish a baseline interface for interoperability between applications. Because FDC3 is largely an agreement between existing platforms and applications, standards should be optimized for ease of adoption rather than functional completeness. Functionality absent from a FDC3 specification is in no way a commentary on its importance.

> The following sections examine the API's use-cases and core concepts. The APIs are fully defined in both subsequent pages of this Part and a full set of TypeScript definitions in the [src](https://github.com/finos/FDC3/tree/main/src/api) directory of the [FDC3 GitHub repository](https://github.com/finos/FDC3/).

Reviewer: Should we go with singular and "The API is fully defined"?

Reviewer 2: Singular is good with me - some refer to the Channel interface as the Channel API, but its really an interface you retrieve from 'the FDC3 API'. There are also other FDC3 APIs (appD and Bridging), but we're within the scope of the FDC3 API (or FDC3 Desktop Agent API).

```suggestion
The following sections examine the API's use-cases and core concepts. The API is fully defined in both subsequent pages of this Part and a full set of TypeScript definitions in the [src](https://github.com/finos/FDC3/tree/main/src/api) directory of the [FDC3 GitHub repository](https://github.com/finos/FDC3/).
```

Reviewer 2: There are a couple more cases of APIs (plural) in this doc, under 'Direct Listening and Broadcast on Channels' and 'Inter-Agent Communication'
